### PR TITLE
Issue 3024: Document format Fixed: controller-service.md

### LINKED ISSUE #3024: Document review : controller-service.md
**Problem description**
 1. Hyperlinks needs to be revised
 2. Format fix

### PR DESCRIPTION
**Pull request by Aparnarr**
**Change log description**  
1. Capitalization of Terms maintained according to Terminology.md
2. Misc HTML tags removed from section headings.
3. Minor content editing performed.

**Purpose of the change**  
Fixes #3024

### CODE REVIEW

**Comment by Aparnarr** [comment]
@shiveshr 
Please review the document!

File: documentation/src/docs/controller-service.md [hunk 1]
```diff
@@ -290,130 +278,131 @@ precisely the key space of the sealed segments. As clients work on
...
-the value by querying zookeeper every time. So we have introduced an
+being processed by same Controller instance, it is suboptimal to read
+the value by querying Zookeeper every time. So we have introduced an
 in-memory cache that each stream store maintains. It caches retrieved
 metadata per stream so that there is maximum one copy of the data per
-stream in the cache. There are two in-memory caches: a) a cache of
-multiple stream objects in the store, b) cache properties of a stream in
-the stream object.
+stream in the cache. There are two in-memory caches: _A cache of
+multiple stream objects in the store_ and _ache properties of a stream in
```
Comment (shiveshr) [inline documentation/src/docs/controller-service.md]: typo: cache and not ache

File: documentation/src/docs/controller-service.md [hunk 2]
```diff
@@ -582,92 +569,88 @@ guarantee at least once processing.
...
 
-#### Update Stream
+### Update Stream
 
-Update stream is implemented as a task on Serialized Request
-Handler/Concurrent Event Processor framework. Update stream is invoked
-by an explicit API call into controller. It first posts an Update
-Request event into request-stream. Following that it tries to create a
+Update stream is implemented as a task on serialized sequest
+handler or concurrent event processor framework. Update stream is invoked
```
Comment (shiveshr) [inline documentation/src/docs/controller-service.md]: ` Update stream is implemented as a task on serialized sequest 
 handler` ~or~over `concurrent event processor framework. Update stream is invoked`

File: documentation/src/docs/controller-service.md (hunk 1 above)
Comment (Aparnarr) [reply documentation/src/docs/controller-service.md]: Accepted suggestion

File: documentation/src/docs/controller-service.md (hunk 2 above)
Comment (Aparnarr) [reply documentation/src/docs/controller-service.md]: Accepted suggestion

**Comment by Aparnarr** [comment]
@shiveshr @fpj 
All comments by reviewer accepted and edited

**Pull request merged**